<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PEI Excel to CSV</string>
  </property>
  <property name="styleSheet">
   <string notr="true">red</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"> </string>
   </property>
   <widget class="QPushButton" name="exitWindow">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ISSUE NO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createCSV">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create CSV files</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="issueNo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseFile">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selectedFile">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
